.github/workflows/main.yml

```diff
@@ -141,7 +141,7 @@ jobs:
         uses: actions/cache@v4
         with:
           path: ~/.cache/Cypress
-          key: binary-${{ env.node-version }}-${{ hashFiles('**/pnpm-lock.yaml') }}
+          key: binary-${{ env.node-version }}-${{ hashFiles('templates/add-ons/frontend/volto-add-on/package.json','templates/add-ons/frontend/volto-add-on/mrs.developer.json') }}
```

Comment:
@wesleybl I think the generated package is in `volto-add-on` in the current working directory (top level), not in `templates/add-ons/frontend/volto-add-on`